CONTRIBUTING.md

```diff
@@ -96,6 +96,24 @@ specific service, for example:
 AWS_REGION=[set region] node node_modules/cucumber/bin/cucumber.js --tag @s3
 ```
 
+### Testing the documentation changes:
+
+To test the documentation changes, please ensure you have the following requirements installed:
+
+* `gem install bundler`
```

Comment:
Perhaps it's worth mentioning that a recent version of Ruby is needed to use gem, but perhaps that's obvious enough.

Reply:
Will still mention it 👍